src/pages/movie/[id].js

```diff
@@ -0,0 +1,144 @@
+import Head from 'next/head';
+import {useRouter} from 'next/router';
+import styled from 'styled-components';
+
+import Layout from '../../components/Layout';
+import useFetch from '../../hooks/useFetch';
+import useStore from '../../hooks/useStore';
+
+const Wrapper = styled.section`
+	padding: 24px;
+`;
+const Picture = styled.img`
```

Comment:
Maybe use a seperate file for all the styling?

Reply:
No. That's the big benefit of having the styling inside the file it is used on. Otherwise i would just use standard css with classes and have multiple options.

Reply:
Okay if you prefer it that way.